site/viamguides/src/pomodoro-bot/pomodoro-bot.md

```diff
@@ -0,0 +1,439 @@
+
+    * Keep the file in a secure location.
+
+### Share Your Calendar with the Service Account
+
+1. Open Google Calendar.
+   ![Google Calendar](assets/googleCalendar.png)
+2. Select the calendar you want to use.
+   ![Select Calendar](assets/selectCalendar.png)
+3. In **Settings and Sharing**, find the "Share with specific people" section:
+   ![Share With Specific People](assets/shareSettings.png)
+    * Enter the email address from the service account key file.
+    * Give it permission to "Make changes to events."
+      ![Share Access](assets/shareAccess.png)
+    * Scroll down to "Integrate Calendar" and copy the **Calendar ID** for the next steps.
+      ![Calendar ID](assets/calendarID.png)
+
+### Connect Your Calendar to the Pomodoro Bot
+
+1. Open the Viam app and go to the **CONFIGURE** tab.
+2. Add the **Calendar module**, and provide:
+   ![Google Calendar Service](assets/addGoogleCalendar.png)
+    * The **Calendar ID** you copied earlier.
+    * The path to the service account JSON file.
+      ![Calendar Configuration](assets/calendarConfig.png)
+3. Save your changes to complete the setup.
+
+### Test the Integration
+
+1. Create a test event in Google Calendar:
+    * Name it something like "Test Meeting" and save the event.
+      ![Test Meeting](assets/testMeeting.png)
+2. Go back to the Viam app and open the **Control** tab.
+      ![Control Tab](assets/controlTab.png)
+3. Use the command interface to check for upcoming calendar events.
+      ![Test Calendar](assets/testCalendar.png)
+
+If everything is working correctly, you should see your test event ("Test Meeting") listed in the results. This confirms that your Pomodoro Bot is successfully linked to your Google Calendar.
+<!-- ------------------------ -->
+## Integrate ENS160 Air Quality Sensor
+
+Duration: 10
+
+Let's integrate the ENS160 air quality sensor with your Pomodoro Bot to monitor indoor air quality effectively. The ENS160 sensor tracks VOCs (volatile organic compounds), eCO2 (equivalent carbon dioxide), and provides an Air Quality Index (AQI) – vital for maintaining an optimal workspace environment.
+
+![ENS160](assets/ens160.png)
+
+### Hardware Setup
+
+* **Power Connections:**
+    * Connect the `VCC` pin of the ENS160 sensor to the `3.3 Volts` pin on the Raspberry Pi.
+    * Connect the `Ground` pin of the ENS160 sensor to a `Ground` pin on the Raspberry Pi.
+
+* **I2C Communication:**
+    * Wire the `SCL` pin of the ENS160 sensor to the `SCL` pin on the Raspberry Pi.
+    * Wire the `SDA` pin of the ENS160 sensor to the `SDA` pin on the Raspberry Pi.
+
+Ensure all connections are secure and double-check your wiring to avoid any issues.
+
+### Software Integration
+
+* **Add the ENS160 Sensor to the Viam App:**
+    1. Open the Viam app.
+    2. Navigate to the **CONFIGURE** tab.
+    3. Add a new component for the ENS160 sensor and wait for the component to finish setup.
+    ![Add ENS160](assets/addENS160.png)
```

Comment:
We should remind people to save their configuration before moving the Control tab since they often forget when walking through instructions:


```suggestion
    3. Add a new component for the ENS160 sensor and wait for the component to finish setup.
    ![Add ENS160](assets/addENS160.png)
    4. Click "Save" to finish the setup. 
```